src/aics-image-viewer/components/App/index.tsx

```diff
@@ -947,7 +961,7 @@ export default class App extends React.Component<AppProps, AppState> {
       };
       // if path trace was enabled in 3D turn it off when switching to 2D.
       if (userSelections[PATH_TRACE]) {
-        this.changeRenderingAlgorithm("volume");
+        this.onChangeRenderingAlgorithm("volume");
```

Comment:
There must be a symbolic constant for `"volume"` here (I hope).  I see PATH_TRACE and MAX_PROJECT used elsewhere.

Reply:
An enum would be an appropriate way to logically encapsulate all 3 modes as a type, too.